definitions/EiffelActivityCanceledEvent/3.2.0.yml

```diff
@@ -0,0 +1,119 @@
+---
+$schema: http://json-schema.org/draft-04/schema#
+_abbrev: ActC
+_description: The EiffelActivityCanceledEvent signals that a previously
+  triggered activity execution has been canceled _before it has started_.
+  This is typically used in queuing situations where a queued execution
+  is dequeued. It is recommended that __CAUSE__ links be used to indicate
+  the reason.
+type: object
+properties:
+  meta:
+    $ref: ../EiffelMetaProperty/3.1.0.yml
+  data:
+    type: object
+    properties:
+      reason:
+        _description: Any human readable information as to the reason
+          for dequeueing.
+        type: string
+      customData:
+        type: array
+        items:
+          $ref: ../EiffelCustomDataProperty/1.0.0.yml
+    additionalProperties: false
+  links:
+    type: array
+    items:
+      $ref: ../EiffelEventLink/1.1.1.yml
+required:
+  - meta
+  - data
+  - links
+additonalProperties: false
+_links:
+  ACTIVITY_EXECUTION:
+    description: Declares the activity execution that was canceled.
+      In other words, [EiffelActivityTriggeredEvent](../eiffel-vocabulary/EiffelActivityTriggeredEvent.md)
+      acts as a handle for the activity execution. This differs from
+      __CONTEXT__. In __ACTIVITY_EXECUTION__ the source carries information
+      pertaining to the target (i.e. the activity started, finished
+      or was canceled). In __CONTEXT__, on the other hand, the source
+      constitutes a subset of the target (e.g. this test case was executed
+      as part of that activity or test suite).
+    required: true
+    multiple: false
+    targets:
+      any_type: false
+      types:
+        - EiffelActivityTriggeredEvent
+  CAUSE:
+    description: While for most events it is recommended that __CAUSE__
+      SHOULD not be used in conjunction with __CONTEXT__, EiffelActivityCanceledEvent
+      is a special case as it represents a deviation from previous
+      intention. Therefore it is recommended that __CAUSE__ always
+      be included where applicable.
+    required: false
+    multiple: true
+    targets:
+      any_type: true
+      types: []
+  CONTEXT:
+    description: Identifies the activity or test suite of which this
+      event constitutes a part.
+    required: false
+    multiple: false
+    targets:
+      any_type: false
+      types:
+        - EiffelActivityTriggeredEvent
+        - EiffelTestSuiteStartedEvent
+  FLOW_CONTEXT:
+    description: 'Identifies the flow context of the event: which is
+      the continuous integration and delivery flow in which this occurred
+      – e.g. which product, project, track or version this is applicable
+      to.'
+    required: false
+    multiple: true
+    targets:
+      any_type: false
+      types:
+        - EiffelFlowContextDefinedEvent
+_history:
+  - version: 3.2.0
+    introduced_in: 'No edition set'
+    changes: Add schema URL to the meta object (see [Issue 313](https://github.com/eiffel-community/eiffel/issues/313)).
```

Comment:
Technically, this should be a link to the _issue_ (280) and not the PR (313). But I can live with this and we can clean it up when updating `introduced_in` as we prepare the release of the Arica edition.

Reply:
If it can help to avoid misunderstanding/confusion, I can fix it on Monday. One sed command can fix that.

Reply:
Issue number fixed. I hope I didn't break something else:-)